package.json

```diff
@@ -130,7 +130,7 @@
     "@types/fs-extra": "^0.0.31",
     "@types/glob": "^5.0.29",
     "@types/jasmine": "^2.2.32",
-    "@types/lodash": "4.14.50",
+    "@types/lodash": "4.14.43",
```

Comment:
`4.14.50` was pinned in https://github.com/angular/angular-cli/pull/4260 recently merged